package.json

```diff
@@ -0,0 +1,41 @@
+{
+  "name": "vinyldns-js",
+  "version": "0.1.0",
+  "description": "javascript client for vinylDNS",
+  "main": "build/vinyldns.bundle.js",
+  "scripts": {
+    "tests": "babel-tape-runner tests/**/*.js | faucet",
+    "prettier": "prettier --print-width 100 --no-bracket-spacing --tab-width 2 --single-quote --write 'src/**/*.js' 'tests/**/*.js'",
+    "webpack": "webpack",
+    "vinylrepl": "sh vinylrepl.sh"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/vinyldns/vinyldns-js.git"
+  },
+  "keywords": [
+    "vinyl",
+    "dns",
+    "vinyldns"
+  ],
+  "license": "Apache-2.0",
+  "bugs": {
+    "url": "https://github.com/vinyldns/vinyldns-js/issues"
+  },
+  "dependencies": {
+    "crypto-js": "^3.1.9-1",
+    "node-fetch": "^2.1.2"
+  },
+  "devDependencies": {
+    "babel-cli": "^6.26.0",
+    "babel-core": "^6.26.0",
+    "babel-loader": "^7.1.4",
+    "babel-preset-es2015": "^6.24.1",
+    "babel-tape-runner": "^2.0.1",
+    "faucet": "^0.0.1",
+    "prettier": "^1.9.2",
+    "tape": "^4.8.0",
+    "webpack": "^4.5.0",
+    "webpack-cli": "^2.0.14"
```

Comment:
I don't believe we'll need babel or webpack, as this NPM module is intended for Node.js environments and not web browsers.

Reply:
Good point since we are forgetting about browser compatibility, but there is a lot of es6 code in this so will need to look into if that will cause problems with node if not using babel